tests/Pager/Doctrine/PagerTest.php

```diff
@@ -13,13 +13,17 @@
 
 namespace Sonata\DatagridBundle\Tests\Pager\Doctrine;
 
+use Doctrine\ORM\AbstractQuery;
 use Doctrine\ORM\Query;
+use Doctrine\ORM\QueryBuilder;
 use PHPUnit\Framework\TestCase;
 use Sonata\DatagridBundle\Pager\Doctrine\Pager;
 use Sonata\DatagridBundle\ProxyQuery\ProxyQueryInterface;
 
 /**
  * @author Romain Mouillard <romain.mouillard@gmail.com>
+ *
+ * @coversDefaultClass \Sonata\DatagridBundle\Pager\Doctrine\Pager
```

Comment:
this can be removed, as we don't us e`@covers` annotations

Reply:
done